src/sonic-config-engine/minigraph.py

```diff
@@ -1505,7 +1505,12 @@ def get_mux_cable_entries(mux_cable_ports, neighbors, devices):
             entry = {}
             neighbor = neighbors[intf]['name']
             entry['state'] = 'auto'
-            entry['server_ipv4'] = devices[neighbor]['lo_addr']
+
+            # Always force a /32 prefix for server IPv4 loopbacks
+            server_ipv4_lo_addr = devices[neighbor]['lo_addr'].split("/")[0]
```

Comment:
can we check if there is prefix length? Are we not converting it to `/32`?

Reply:
This code is agnostic of existing prefix length and behaves the same with or without. The call to `ip_network` below implicitly adds a prefix length of /32, since we have removed any existing prefix here.